src/node/eviction.cpp

```diff
@@ -243,6 +244,53 @@ void ProtectEvictionCandidatesByRatio(std::vector<NodeEvictionCandidate>& evicti
 EvictionManagerImpl::EvictionManagerImpl() {}
 EvictionManagerImpl::~EvictionManagerImpl() = default;
 
+void EvictionManagerImpl::AddCandidate(NodeId id, std::chrono::seconds connected,
+                                       uint64_t keyed_net_group, bool prefer_evict,
+                                       bool is_local, Network network,
+                                       bool noban, ConnectionType conn_type)
+{
+    LOCK(m_candidates_mutex);
+    // The default values used to initialise the eviction candidates result in
```

Comment:
Why don't go further and summarize which parameters actually matter? Or just mention `why` it is so

Reply:
Added some comments for these values.